packages/runtime/src/swagger/swagger.ts

```diff
@@ -217,7 +217,7 @@ export namespace Swagger {
 
   export interface Response3 {
     description: string;
-    content?: { [name: string]: Schema & Example };
+    content?: { [name: string]: Schema3 & { schema?: Schema } & Example };
```

Comment:
can schema really be optional? It's as optional everywehre else, therefore I did it here too.. but not sure if needed

Reply:
Maybe it is simply related to partial responses being built?
I don't really know an example where there wouldn't be a schema

Reply:
I've addressed this in a second PR https://github.com/lukeautry/tsoa/pull/1253 
schema is not optional there anymore, as there is not situation where this can be the case